src/holdingpen/components/ExceptionsTable.jsx

```diff
@@ -0,0 +1,170 @@
+import React, {Component} from 'react';
+import { Table, Input, Button, Icon } from 'antd';
+
+class exceptionsTable extends Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      filteredInfo: null,
+      data: props.exceptions,
+      record: {
+        filterDropdownVisible: false,
+        searchText: '',
+        filtered: false,
+      },
+      error: {
+        filterDropdownVisible: false,
+        searchText: '',
+        filtered: false,
+      }
+    }
+  }
+
+  onInputChange = (e, key) => {
+    const keyToUpdate = {...this.state[key]};
+    keyToUpdate.searchText = e.target.value;
+    this.setState({
+      [key] : keyToUpdate,
+    })
+  }
+
+  onSearch = (key) => {
+    const exceptions = this.props.exceptions;
+    const keyToUpdate = {...this.state[key]};
+    const searchText = keyToUpdate.searchText;
+    const reg = new RegExp(searchText, 'gi');
+    keyToUpdate.filterDropdownVisible = false;
+    keyToUpdate.filtered = !!searchText;
+    this.setState({
+      [key]: keyToUpdate,
+      data: exceptions.map((exception) => {
+        const match = exception[key].match(reg);
+        if (!match) {
+          return null;
+        }
+        return {
+          ...exception,
+          [key]: (
+            <span>
+              {exception[key].split(new RegExp(`(?<=${searchText})|(?=${searchText})`, 'i')).map((text, i) => (
+                text.toLowerCase() === searchText.toLowerCase() ?
+                  <span key={i} className="highlight">{text}</span> : text // eslint-disable-line
+              ))}
+            </span>
+          ),
+        };
+      }).filter(exception => !!exception),
+    });
+  }
+  
+  handleChange = (pagination, filters) => {
+    this.setState({
+      filteredInfo: filters,
+    });
+  }
+  
+  clearFilters = () => {
+    this.setState({ filteredInfo: null });
+  }
+
+  render() {
+    let { filteredInfo } = this.state;
+    filteredInfo = filteredInfo || {};
+
+    const columns = [{
+      title: 'Collection',
+      dataIndex: 'collection',
+      key: 'collection',
+      filters: [
+        { text: 'HEP', value: 'HEP' },
+        { text: 'HEPNAMES', value: 'HEPNAMES' },
+        { text: 'JOB', value: 'JOB' },
+        { text: 'JOBHIDDEN', value: 'JOBHIDDEN' },
+        { text: 'CONFERENCES', value: 'CONFERENCES' },
+        { text: 'JOURNALS', value: 'JOURNALS' },
+      ],
+      filteredValue: filteredInfo.collection || null,
```

Comment:
this looks weird, because null is also falsy, so you're checking if `filteredInfo.collection` is falsy then assign `null` but you could just remove `|| null` part, unless you have `=== null`